<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>param_sensitivity_widget</class>
 <widget class="QWidget" name="param_sensitivity_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Realizations</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="realizationsListWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Responses</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QScrollArea" name="parameterScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>355</width>
        <height>76</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QListWidget" name="selectedParamListWidget"/>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="plotPushButton">
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTreeView" name="responseTreeView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
